src/entitykeys.js

```diff
@@ -99,6 +100,105 @@ exports.StringKey = function(key) {
   };
 };
 
+/******************** Descending Integer Key ********************/
+
+// More nines than an int can hold, ie. MORE_NINES_THAN_INT > 2^32
```

Comment:
Maybe add here that this `2^32` is being enforced at https://github.com/taskcluster/azure-entities/pull/18/files#diff-13faf894bb1c5f4395134e024024824fR267

Otherwise it's a bit confusing since js doesn't have that limit.